app/config/__init__.py

```diff
@@ -22,6 +24,49 @@ class DatabaseSecret(BaseModel):
     password: str
 
 
+FS_CONTENT_SECURITY_POLICY = {
+    "default-src": ["'self'"],
+    "script-src": ["'self'"],
+    "img-src": ["'self'"],
+    "style-src": ["'self'"],
+}
+
+
+def make_development_csp() -> dict[str, list[str]]:
+    csp = copy.deepcopy(FS_CONTENT_SECURITY_POLICY)
+    csp["default-src"].extend(
+        [
+            "http://localhost:5173",  # Vite assets
```

Comment:
Can this come from `app.config["ASSETS_VITE_BASE_URL"]`? Maybe needs a bit of thinking about how it can sensible build it for http and ws but could always have a URL and WSS config 

Probably not a deal breaker but something that might be a pain to track down for a port change or something like that